pyproject.toml

```
@@ -4,7 +4,7 @@ version = "2.0.1"
 description = "Python dependency management and packaging made easy."
 requires-python = ">=3.9,<4.0"
 dependencies = [
-    "poetry-core @ git+https://github.com/python-poetry/poetry-core.git",
+    "poetry-core @ git+https://github.com/radoering/poetry-core.git@perf-marker-intersection",
```

```suggestion
    "poetry-core @ git+https://github.com/poetry-core/poetry-core.git@main",
```

python-poetry/poetry-core#821 is now merged.